### PR TITLE
Resolve splats in base role

### PR DESCRIPTION
**Pull request by PanSpagetka**
When base role permissions contains `*`and respected permissions are not at first page, no permission are preselected. 

This is starting to be crazy, needs to simplify the logic.

Issue 10: https://issues.redhat.com/browse/RHCLOUD-9983

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@PanSpagetka :crossed_swords:

**Comment by Hyperkid123** [comment]
@PanSpagetka I think there is a commit that does not belong here.

File: src/redux/actions/permission-action.js [hunk 1]
```diff
@@ -16,3 +16,25 @@ export const listPermissionOptions = ({ field, limit, offset, application, resou
   type: fieldToAction[field],
   payload: PermissionsHelper.listPermissionOptions(field, limit, offset, application, resourceType, verb, options),
 });
+
+export const expandSplats = ({
+  limit = 1000,
```
Comment (Hyperkid123) [inline src/redux/actions/permission-action.js:21]: does this endpoint ignore the max API limit of 100?

Comment (PanSpagetka) [reply src/redux/actions/permission-action.js:21]: When I tried it in API docs, it seems that 1000 is the limit.

Comment (Hyperkid123) [reply src/redux/actions/permission-action.js:21]: Nice good to know :+1:

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**